Engine/source/math/mMatrix.h

```diff
@@ -257,6 +275,14 @@ inline MatrixF& MatrixF::set( const EulerF &e, const Point3F& p)
    return (*this);
 }
 
+inline bool MatrixF::equals(const MatrixF& mat)
+{
+   return (m[0] == mat[0] && m[1] == mat[1] && m[2] == mat[2] && m[3] == mat[3] &&
```

Comment:
Same as operator==. Could be called by operator== instead of unnecessarily duplicating.